src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,24 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private int year;
```

Comment:
The field 'year' should have an access modifier to encapsulate the field properly. Consider using 'private' to ensure that direct access is restricted and use getters and setters for access.